source/retryable-reads/retryable-reads.rst

```diff
@@ -471,7 +471,7 @@ for tests.
 
 At a high level, the test plan will cover executing supported read operations
 within a MongoClient where retryable reads have been enabled, ensuring that
-reads are retried exactly once.
+reads are retried.
```

Comment:
These tests contradict the "retry once" behavior outlined above. For example, the abstract reads,

> This specification is about the ability for drivers to automatically retry any read operation that has not yet received any results—due to a transient network error, a "not master" error after a replica set failover, etc.—**exactly once**.

The pseudocode also shows a single retry. This spec should be updated too. Otherwise, Drivers will sync new tests, have those tests fail, and only have incorrect guidance on how retryable reads are supposed to behave.

Reply:
Yes. Could you add a reference to it in your top comment? I guess I'll hold off on this until that PR is reviewed since changes in #881 could cause downstream changes.

Reply:
Done.